qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/Test/CVforNotOpenPosition.feature

```diff
@@ -0,0 +1,122 @@
+Background: User went to the "Send us your CV" form at the down of the page 
+
+Feature: Send CV for not open job position  
+
+@Happy_pass
+Scenario: Send CV happy pass
+Given User entered valid Name
+And User entered valid e-mail
+And User checked the checkbox
+And User attached CV in .pdf file 10 Mb
+When User press send button
+Then the CV should be received by recruiters
+
+Scenario: Send CV with message happy pass
+Given User entered valid Name
+And User entered valid e-mail
+And User entered 500 symbols in message field
+And User checked the checkbox
+And User attached CV in .docx file 10 Mb
+When User press send button
+Then the CV should be received by recruiters
+
+@negative_tests
+
+Scenario: Send CV with too large message 
+Given User entered valid Name
+And User entered valid e-mail
+And User entered 501 symbols in message field
+And User checked the checkbox
+And User attached CV in .docx file 10 Mb
+When User press send button
+Then CV should be unsend, mistake message about lenght message no more than 500 symbol
+
+Scenario: Empty forms, correct attached file
+Given User tipped no symbols in the name
+And User tipped no symbols in the e-mail
+And User attached CV in .pdf file
+And User checked the checkbox
+When User press send button
+Then CV should be unsend, all empty field should be highlighted in red
+
+Scenario: No CV attached
+Given User entered valid Name
+And User entered valid e-mail
+And User checked the checkbox
+When User press send button
+Then CV should be unsend, attached CV button should be highlighted in red
+
+Scenario: Unchecked checkbox 
+Given User entered valid Name
+And User entered valid e-mail
+And User attached CV in .pdf file
+When User press send button
+Then CV should be unsent, checkbox should be highlighted in red
+
+@specific_tests_for_some_forms
+Scenario Outline: format file testing
+Given User entered valid Name
+And User entered valid e-mail
+And User checked the checkbox 
+And User attached CV in <Format>
+And User attached CV <> Mb
+When User press send
+Then the CV should be <Result> 
```

Comment:
It should be better to check file formats before pressing send button. In this case, if file format is valid, we will see specific text bellow attach button (name of attach file). If format is invalid, modal window with specific error text will be opened.

Reply:
Oh! Of cource! Thank you very much! It is really rude mistake ((( I have to try to attached files in incorrect format!!!!! Corrected.